app/javascript/styles/_index.scss

```diff
@@ -1002,7 +1002,7 @@ html {
 }
 
 // For when a user is signed in
-.govuk-navigation-header {
+.fb-navigation-header {
```

Comment:
Renamed this class, as it is not a design system class (our header differs from the design system)  

Figured that it would help avoid confusion if the prefix was `fb` to indicate it is a formbuilder/mojforms class.

There's more to do along these lines.  But restricted it to just this one in this PR.